visitor-app/src/visitors/pages/SignUp/SignUpForm/sideCopy.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
If this is a component the filename should be `PascalCase`

Reply:
We're not following this convention in this repo. Happy to move forward like this though.